### PR TITLE
Refactor the 'Leadership vs Management' explanation

### PR DESCRIPTION
**Pull request by dobromirmontauk**
Larmon pointed out that just defining Leadership vs Management has different approaches; hence, refactoring this page to use Recipes for this separation.

### CODE REVIEW

File: recipes/README.md [hunk 1]
```diff
@@ -4,6 +4,18 @@
...
+
+
+# Explaining the Difference
+
+We've seen the question "what is the difference between Leadership & Management?" come up often enough that it deserves its own set of recipes. Please contribute your own explanation here, or link to really good ones you find online!
+
+## Dobromir's Recipe
 
 Both Leadership & Management are fundamentally about changing how people think. Change how they think, and they'll
 behave differently. Change how they behave, and they'll reach different outcomes. Like the physics equivalents of
```
Comment (joannakrk) [inline recipes/README.md:21]: this file has a different line break than the other ones - is this something you want to fix? (that is - you have lines broken mid-paragraph here, while the other docs keep single-line sentences... )

File: recipes/README.md [hunk 2]
```diff
@@ -121,3 +133,10 @@ about being perfect, we haven't met anybody who's achieved perfection in either,
 
 Hopefully, this cookbook will give you recipes where you need them most. 
 
+
+## TODO
+My recipe has a TON of stuff I want to expand on. Notes here:
+
+1. My L/M recipe: leadership is about risk to yourself, to your reputation, to your career. If there is no risk, it’s not leadership. 
+2. Small leadership ‘bumps’ for a Sr. EM might be huge leadership moments for more junior employees. That’s why leadership/management is a spectrum. 
+3. See all my notes I wrote up in Google Docs [here](https://docs.google.com/document/d/1LwOIhMI0MsCYePKU7sz2eigs-PcSJ6tuVXIbZGtUJdk/edit#heading=h.gu4ti1e72vvr). Need to integrate that here.
```
Comment (joannakrk) [inline recipes/README.md:142]: Could you add here "strong leader / bad manager and poor leader / strong manager explorations" so we don't forget to mention it?

**Review by joannakrk — APPROVED** [review]

File: recipes/README.md (hunk 1 above)
Comment (dobromirmontauk) [reply recipes/README.md:21]: I'm going to run scripts to clean the formatting up in the future so I'm not worrying about it yet.

File: recipes/README.md (hunk 2 above)
Comment (dobromirmontauk) [reply recipes/README.md]: Oh yes thank you, added.

**Pull request merged**